tests/test_display.py

```diff
@@ -534,7 +521,7 @@ def test_sharex_specshow_ms(S_abs, y, sr):
     return plt.gcf()
 
 
-@pytest.mark.mpl_image_compare(baseline_images=['sharex_waveplot_ms'], extensions=['png'])
+@pytest.mark.mpl_image_compare(baseline_images=['sharex_waveplot_ms'], extensions=['png'], tolerance=6)
```

Comment:
increased RMS tolerance here is to handle deviations induced by different versions of the freetype library.  It's still small enough tolerance that actual failures should trigger, but now we can at least have minor changes in axis label kerning without triggering a failure.